Public/Src/Tools/Execution.Analyzer/Analyzers.Partner/DumpPipAnalyzer.cs

```diff
@@ -505,6 +505,13 @@ private XElement GetProcessDetails(Process pip)
                     m_html.CreateRow("Untracked Paths", pip.UntrackedPaths),
                     m_html.CreateRow("Untracked Scopes", pip.UntrackedScopes)),
 
+                m_html.CreateBlock(
+                    "Global Dependencies",
+                    (pip.RequireGlobalDependencies && m_invocationData.Configuration.Sandbox.GlobalUnsafePassthroughEnvironmentVariables != null ? 

```

Comment:
Should we only present the values when the pip opt them in?

Reply:
HtmlHelper handles that